bench/README.md

```diff
@@ -42,14 +41,15 @@ By that reason load test uses prepared load data which can be generated as follo
 *   Generate test transactions:
 
     ```bash
-    # DCLBENCH_WRITE_USERS_COUNT=<NUM-USERS> DCLBENCH_WRITE_USERS_Q_COUNT=<NUM-REQ-PER-USER> python bench/generate.py bench/test.spec.yaml bench/txns
+    # DCLBENCH_WRITE_USERS_COUNT=<NUM-USERS> DCLBENCH_WRITE_USERS_Q_COUNT=<NUM-REQ-PER-USER> DCLBENCH_ADD_NEW_MODELS_COUNT=<NUM-NEW-MODELS> python bench/generate.py bench/test.spec.yaml bench/txns
     python bench/generate.py bench/test.spec.yaml bench/txns
     ```
 
 Here the following (**optional**) inputs are considered:
 
 *   `NUM-USERS`: number of client accounts with write access (created as Vendors). Default: 10
 *   `NUM-REQ-PER-USER`: number of write txns to perform per a user. Default: 1000
+*   `NUM-NEW-MODELS`: models are added to the ledger for reading afterwards in load tests. Default: 5
 
 ## Run
 
```

Comment:
It would be great to provide explicit documentation how to:
1) Run read-only requests
2) Run write-only requests
3) Run mixed mode (write and read)